<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>20</y>
     <width>445</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label_inputstring">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>13</y>
      <width>75</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;输入字符串&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_inputstring">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>171</width>
      <height>25</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_foreach">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>10</y>
      <width>145</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>每个字符的Unicode</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_getitem">
    <property name="geometry">
     <rect>
      <x>288</x>
      <y>50</y>
      <width>144</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>单个字符的特性判断</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_inputchar">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>51</y>
      <width>171</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>50</y>
      <width>90</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;输入单个字符&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>110</y>
     <width>441</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>其他测试和功能</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pushButton_test1">
      <property name="text">
       <string>PushButton_test1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_test2">
      <property name="text">
       <string>PushButton_test2</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_check">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>220</y>
     <width>441</width>
     <height>331</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>291</width>
      <height>311</height>
     </rect>
    </property>
    <property name="title">
     <string>显示结果</string>
    </property>
    <widget class="QTextEdit" name="textEdit_outputs">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>261</width>
       <height>261</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>20</y>
      <width>91</width>
      <height>291</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QCheckBox" name="checkBox_isDigit">
       <property name="text">
        <string>isDigit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_isLetter">
       <property name="text">
        <string>isLetter</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_isUpper">
       <property name="text">
        <string>isUpper</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_isLower">
       <property name="text">
        <string>isLower</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_isMark">
       <property name="text">
        <string>is_Mark</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_test2</sender>
   <signal>clicked()</signal>
   <receiver>Widget</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>506</x>
     <y>164</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
